translate/src/modules/notification/components/BadgeTooltip.css

```diff
@@ -0,0 +1,32 @@
+.badge-tooltip {
+  background: var(--black-3);
+  border: 1px solid var(--dark-grey-1);
+  border-radius: 10px;
+  box-shadow: 0 0 20px -2px var(--tooltip-background);
+  box-sizing: border-box;
+  text-align: center;
+  margin: 0;
+  padding: 20px;
+  position: absolute;
+  top: 300px;
+  left: 200px;
+  width: 350px;
+  z-index: 100;
+}
+
+/* Showing and hiding animations. */
+.badge-tooltip.showing {
+  top: 0;
+}
+
+.badge-tooltip button {
+  background: none;
+  border: none;
+  color: var(--tooltip-color-2);
+  font-weight: 300;
+}
```

Comment:
Eemeli would suggest that we use CSS nesting in the new files: 😄
https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_nesting/Using_CSS_nesting